patch/QuadraphonicMixer/QuadraphonicMixer.cpp

```diff
@@ -0,0 +1,314 @@
+#include "daisysp.h"
+#include "daisy_patch.h"
+#include <string>
+#include <limits>
+#include <vector>
+#include "quadMixer.h"
+#include "stereoMixer.h"
+
+using namespace daisy;
+using namespace daisysp;
+using namespace QuadraphonicMixer;
+
+DaisyPatch patch;
+float sampleRate;
+constexpr uint8_t NUM_CTRL = 4;
+float ctrl[4];
+constexpr uint8_t NUM_GATE = 2;
+bool gate[NUM_GATE];
+constexpr uint8_t NUM_CHANNEL = 4;
+constexpr float ONE_ONEHUNDREDTH = 0.01f;
+
+enum State : uint8_t
+{
+    ParamSelect = 0,
+    Stereo = 1,
+    ReverbWet = 2,
+    ReverbFeedback = 3,
+    CH1Gain = 4,
+    CH2Gain = 5,
+    CH3Gain = 6,
+    CH4Gain = 7
+};
+constexpr uint8_t NUM_STATES = 8;
+constexpr uint8_t NUM_PARAM = NUM_STATES - 1;
+State currState = State::ParamSelect;
+uint8_t currParam = 0;
+
+bool stereo = true;
```

Comment:
curious if folks think stereo should be the default behavior vs quad. Personally, I use stereo mode more than quad.

Reply:
Hmm, stereo seems more generically useful. Quad seems cooler. Also, based on the name of the example I would expect it to be quad by default.